### PR TITLE
Issue one request per uploaded file

### PR DESCRIPTION
**Pull request by drdavella**
While this isn't the most elegant solution, it helps us avoid some HTTP 413 errors when uploading large numbers of Sonar issues/hotspots. Later we might wish to upload compressed inputs in one form or another.

### CODE REVIEW

File: dist/index.js
Comment (drdavella) [inline dist/index.js:1]: This was needed for testing and should be removed before merging.

File: src/pixee-platform.ts [hunk 1]
```diff
@@ -6,29 +6,27 @@ import { TOOL_PATH } from "./inputs";
...
-  // Append each file to the form data
-  files.forEach((file) => {
-    form.append("files", fs.readFileSync(file), path.basename(file));
-  });
-
   const pixeeUrl = core.getInput("pixee-api-url");
   const token = await core.getIDToken(pixeeUrl);
   const url = buildUploadApiUrl(tool);
 
-  return axios
```
Comment (gilday) [inline src/pixee-platform.ts:22]: The action module expects to `await` on the promise returned by this method. This change eliminates the returning of the promise.

You'll want to flatten all the promises from each request into one promise and return that.

**Review by gilday — CHANGES_REQUESTED** [review]

File: dist/index.js
Comment (drdavella) [inline dist/index.js:1]: This is necessary for testing and should be removed before merging.

File: src/pixee-platform.ts [hunk 2]
```diff
@@ -6,29 +6,34 @@ import { TOOL_PATH } from "./inputs";
...
+        },
+      })
+      .then(() => {
+        // don't return the axios response
+        console.log(`Uploaded ${file} to ${url}`);
+      })
+      .catch((error) => {
+        console.error(`Failed to upload ${file} to ${url}`);
+        console.error(error);
+      });
```
Comment (gilday) [inline src/pixee-platform.ts:33]: Is the expected behavior that any upload failure doesn't prevent other uploads?

**Review by gilday — APPROVED** [review]

File: src/pixee-platform.ts (hunk 2 above)
Comment (drdavella) [reply src/pixee-platform.ts:33]: Probably it's better if it just fails the whole job. I'll fix it.

**Pull request merged**